pydantic/types.py

```diff
@@ -606,3 +607,68 @@ def _get_brand(card_number: str) -> PaymentCardBrand:
         else:
             brand = PaymentCardBrand.other
         return brand
+
+
+BYTE_SIZES = {
+    '': 1,
+    'B': 1,
+    'kB': 10 ** 3,
+    'MB': 10 ** 6,
+    'GB': 10 ** 9,
+    'TB': 10 ** 12,
+    'PB': 10 ** 15,
+    'EB': 10 ** 18,
```

Comment:
I'm not sure about `EB`, it's pretty rare. If we go this far we could also add Zetta and Yotta.

Maybe best to stop at `PB` which everyone knows?

I'm not sure myself.

Reply:
We sometimes deal with EB (high energy physics) which is why I had it in, but my perspective is likely a bit skewed.

Reply:
let's keep it then.